lib/class-wp-rest-customizer-nonces.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * WP_Rest_Customizer_Nonces class.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Class that returns the customizer "save" nonce that's required for the
+ * batch save operation using the customizer API endpoint.
+ */
+class WP_Rest_Customizer_Nonces extends WP_REST_Controller {
```

Comment:
Doesn't apiFetch handle nonces?

Reply:
It does handle a very specific `rest-api` nonce. This nonce is different as it depends on the current theme. I didn't see a clean way to support it in the existing nonce machinery so I added this endpoint until we're able to do "proper" batch requests.

Reply:
Got it!